lightningd/pay.c

```diff
@@ -890,6 +890,98 @@ param_route_hop(struct command *cmd, const char *name, const char *buffer,
 	return NULL;
 }
 
+static struct command_result *json_sendonion(struct command *cmd,
+					     const char *buffer,
+					     const jsmntok_t *obj UNNEEDED,
+					     const jsmntok_t *params)
+{
+	u8 *onion;
+	struct onionpacket *packet;
+	enum onion_type failcode;
+	struct htlc_out *hout;
+	struct route_hop *first_hop;
+	struct sha256 *payment_hash;
+	struct channel *channel;
+	struct lightningd *ld = cmd->ld;
+	struct wallet_payment *payment;
+	const char *label;
+
+	if (!params || !deprecated_apis) {
```

Comment:
This test should be eliminated?  Looks like cut & paste from a function which had to handle weird deprecated options?